### PR TITLE
rst-1067-acas-admin

### PR DESCRIPTION
**Pull request by sivvoy**

### CODE REVIEW

File: features/end_to_end/acas_certificate.feature [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@javascript
+Feature:
+  As an ACAS administrator
+  I want to download a certificate/document from the ET Admin
+  So I can validate the certificate number and its contents
+  And view logs detailing user access and error messages
+
+  Scenario: Download and view ACAS Certificate
+    Given I am an ACAS administrator
+    When I enter an ACAS certificate number
```
Comment (AlexaVSmith) [inline features/end_to_end/acas_certificate.feature]: When I enter an ACAS certificate number <-- where? 
What about:
When I enter an ACAS certificate number into the ET Admin

File: features/end_to_end/acas_certificate.feature [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Then I can download and view the contents of the document
+
+  Scenario: Certificate not found
+    Given I am an ACAS administrator
+    When I enter an invalid ACAS certificate number
```
Comment (AlexaVSmith) [inline features/end_to_end/acas_certificate.feature:16]: As above

File: features/end_to_end/acas_certificate.feature [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    Then the system should return 'No certifciate returned from ACAS for AC123456/78/90'
+
+  Scenario: Invalid format
+    Given I am an ACAS administrator
+    When I enter an invalid format ACAS number
```
Comment (AlexaVSmith) [inline features/end_to_end/acas_certificate.feature]: As above

File: features/end_to_end/acas_certificate.feature [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  Scenario: ACAS log
+    Given I am an ET Administrator with full privileges 
+    When I view the ACAS Certificate Access log
+    And sort by the Date field
+    Then I can see who has downloaded the ACAS document
+    
+  Scenario: No response from ACAS
+    Given I am an ACAS administrator
+    When I enter an ACAS certificate number
+    And an ACAS server is down for unknown reason
```
Comment (AlexaVSmith) [inline features/end_to_end/acas_certificate.feature]: for unknown reason -- > for an unknown reason

**Review by AlexaVSmith — CHANGES_REQUESTED** [review]
Very small suggested changes.

File: features/end_to_end/acas_certificate.feature (hunk 1 above)
Comment (sivvoy) [reply features/end_to_end/acas_certificate.feature]: When I enter an ACAS certificate number in the ACAS search field?

**Review by garytaylor — APPROVED** [review]
Looks good to me

**Review by AlexaVSmith — APPROVED** [review]

**Pull request merged**